### ISSUE TITLE
[HW1] Python HTMLParser library 

### ISSUE THREAD

**Issue by Evan-JH-Kim** [swsnu/swppfall2018#4]
HW1 Python 과제에서 html parsing을 위해 HTMLParser library를 import하여도 되나요?

**Comment by insung151** [comment]
ㅖ

**Comment by insung151** [comment]
@yuyupopo
다른 어떤 라이브러리도 import해서는 안되나요?
ex) import os

**Comment by sanha** [comment]
@insung151 os는 import 해야겠네요. import 추가해서 skeleton code를 update하겠습니다.

**Comment by sanha** [comment]
@Evan-JH-Kim glob이 크게 필요할 것 같지 않은데, 어떤 용도로 사용하시나요?

**Comment by Evan-JH-Kim** [comment]
path를 가져온 후 디렉터리에 있는 모든 파일들을 리스트로 변환하려 합니다.

**Comment by Evan-JH-Kim** [comment]
check_filename_existence 함수가 현재 directory 안에 argument로 준 이름에 해당하는 file이 있는지 체크해야 하는 것 아닌가요?

**Comment by yuyupopo** [comment]
네 file existence를 체크하는 것은 맞습니다. 다만 폴더 내의 모든 파일들을 확인하는 것은 필요 없어보입니다.
